wavelink/player.py

```diff
@@ -241,11 +241,15 @@ async def hook(self, event) -> None:
 
     def _get_shard_socket(self, shard_id: int) -> Optional[DiscordWebSocket]:
         if isinstance(self.bot, commands.AutoShardedBot):
-            return self.bot.shards[shard_id].ws
+            if hasattr(self.bot.shards[shard_id],"ws"):
+                return self.bot.shards[shard_id].ws
+            else:
+                return self.bot.shards[shard_id]._parent.ws
```

Comment:
```suggestion
            try:
                return self.bot.shards[shard_id].ws
            except AttributeError:
                return self.bot.shards[shard_id]._parent.ws
```
Would be more ideal.